.prow/verify.yaml

```diff
@@ -65,7 +65,7 @@ presubmits:
       preset-goproxy: "true"
     spec:
       containers:
-        - image: quay.io/kubermatic/wwhrd:0.4.0-2
+        - image: quay.io/kubermatic/build:go-1.19-node-18-kind-0.14-0
```

Comment:
Why is this image change needed?

Reply:
This image was using Go 1.16 and was thus failing https://public-prow.loodse.com/view/gs/prow-dev-public-data/pr-logs/pull/kubermatic_kubeone/2367/pull-kubeone-license-validation/1573160617416593408